python/tests/test_vars_funs.py

```diff
@@ -0,0 +1,185 @@
+import pandas as pd
```

Comment:
These tests are largely adapted from [the corresponding R tests](https://github.com/ccao-data/ccao/blob/d85e0d4729b12f1bd2bc200b0b8a93ccca5cdd38/tests/testthat/test-vars_funs.R#L38-L189), although I removed some tests that are no longer useful (in particular ones that test the legacy vars dict) and added a couple more for completeness.